testsuite/testsuite-buildtool/.gitignore

```diff
@@ -0,0 +1,203 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
Can we not have a .gitignore deep in the tree, and just promote anything reasonable to the root .gitignore?